### PR TITLE
#12 modify the code for the pipeline

### PR DESCRIPTION
**Pull request by mmbc2008**

### CODE REVIEW

File: src/opinion_summarisation/models/train.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+else:
+    device = "cpu"
+
+train_df = pd.read_csv('/content/drive/MyDrive/Colab Notebooks/BachelorsProject/euans_reviews.csv')
+
+test_df = pd.read_csv("/content/drive/MyDrive/Colab Notebooks/BachelorsProject/google_reviews.csv")
+
+euans_reviews = train_df.Text.values.tolist()
+
+import time
```
Comment (igornishka) [inline src/opinion_summarisation/models/train.py:30]: imports on top -> split into a few functions -> clean up (whole file)

**Review by igornishka — COMMENTED** [review]
Also check the issues from [here](https://github.com/Amsterdam-Internships/Venue-Accessibility-Google-Reviews/issues/27):
- [ ] update readme with setting up and usage instructions (.env + paths + scripts + args)
- [ ] update steps -> make_dataset -> train -> ???
- [ ] clean up / document dataset making/cleaning - no need for the 1line functions, improve naming (e.g. select aspects?!, removing nans does more than removing nans, etc)
- [x] no spaces in folder/file names (in processed)

File: media/Fonts/Roboto/Roboto-Black.ttf
[the remark this answers is not in the document]
Comment (mmbc2008) [reply media/Fonts/Roboto/Roboto-Black.ttf:1]: These are no longer needed.

File: .gitignore [hunk 2]
```diff
@@ -1,14 +1,29 @@
 # env / packaging
-env/
```
[the remark this answers is not in the document]
Comment (mmbc2008) [reply .gitignore:2]: This folder isn’t in the repo anymore so it was probably just something I forgot to delete.

File: notebooks/__init__.py
[the remark this answers is not in the document]
Comment (mmbc2008) [reply notebooks/__init__.py:1]: I have removed this.

File: src/opinion_summarisation/models/train.py (hunk 1 above)
Comment (mmbc2008) [reply src/opinion_summarisation/models/train.py:30]: This will be moved to a different script altogether.

**Comment by mmbc2008** [comment]
Have fixed everything except pylint

**Pull request merged**